DynamicActiveRecord.php

```diff
@@ -44,22 +44,43 @@
  */
 class DynamicActiveRecord extends ActiveRecord
 {
-    /**
-     * Prefix for base64 encoded dynamic attribute values
-     */
-    const DATA_URI_PREFIX = 'data:application/octet-stream;base64,';
+    private $_dynamicAttributes = [];
+
+    protected static $_encoder;
```

Comment:
I wanted to make it private but it doesn't work, probably because it's static and we refer to it through `static::$_encoder`.